setup.cfg

```diff
@@ -33,7 +33,10 @@ install_requires =
 
 [options.extras_require]
 duecredit = duecredit
-test = coverage<5
+test =
+    coverage<5
```

Comment:
Can we update coverage at this point? Would you mind making a new branch with this where we use latset?